src/coordinator/api/v1/handler/database/create.go

```diff
@@ -54,11 +54,13 @@ const (
 
 	idealDatapointsPerBlock           = 720
 	blockSizeFromExpectedSeriesScalar = idealDatapointsPerBlock * int64(time.Hour)
+	shardMultiplier                   = 32
 
 	dbTypeLocal                 dbType = "local"
+	dbTypeGroup                 dbType = "group"
```

Comment:
maybe just `dbTypeCluster`?

Reply:
Sure that works. Only caveat is that this endpoint puts everything into the same zone _and_ isolation group. Not sure if we'd want a separate type so that users can specify different racks, etc.

Reply:
Hm good catch, I see what you mean. Maybe take two slices for hostnames - one for `[]string` and the other for `[]struct{string(name), string(isolation group)}`. Only one can be defined at a time, and that way users can use the same endpoint for both use-cases. What do you reckon?